partner_restrict_payment_acquirer/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    allowed_acquirer_ids = fields.Many2many(
+        comodel_name="payment.acquirer",
+        string="Allowed Acquirers",
+        domain=[("state", "in", ["enabled", "test"])],
```

Comment:
Would be good to put relational table explicitly here. In case some other module adds similar relation:
```
relation="partner_acquirere_allowed_rel",
column1="partner_id",
columnt2="acquirer_id"
```